### DISCUSSION TITLE
grpc and web_socket_channel package updates

### DISCUSSION THREAD

**Discussion by lehmannk** [fzyzcjy/flutter_convenient_test#406] [Ideas]
can you make a release with grpc package ^4.0.0 and web_socket_channel package ^3.0.0? I run into build issues with the current version.

**Comment by fzyzcjy** [comment]
Sure, feel free to PR!

**Reply by lehmannk** [reply]
there is a build error for mac, do you know how to handle this? I only have a windows PC:
https://github.com/fzyzcjy/flutter_convenient_test/actions/runs/11360030361

**Reply by fzyzcjy** [reply]
(replied there)

**Reply by lehmannk** [reply]
ok, all fixed, PR is ready
